addons/mail/static/src/messaging/entity/activity/activity.js

```diff
@@ -100,7 +105,7 @@ function ActivityFactory({ Entity }) {
                     creatorDisplayName,
                 ] = [],
                 date_deadline: dateDeadline,
-                forceNext = this.forceNext || false,
+                force_next: forceNext = this.forceNext || false,
```

Comment:
This change does not seem correct, it should be all of them snake case or all of them camel case.
The object assign is using the value from forceNext too.